packages/appeals-service-api/__tests__/unit/controllers/token.test.js

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
FYI - I'd written these test stubs before realising the unit tests in appeals-service-api are configured not to run. I've left them in this commented-out state in case this changes in future, it seemed a shame to throw away the work. But I also understand that it could be considered messy. So please let me know if I should remove these.

Reply:
it would be good to get this functionality tested - is there any way to configure only these specific tests to run (given the large number of outdated unit tests in the unit test folder)

Reply:
I've updated this now with completed unit tests for the code I added to `appeals-service-api`, moved the old/obsolete unit tests to a new directory `__legacy-tests__` and reconfigured jest to run both developer and unit tests.